### ISSUE TITLE
RealtimeDatabase and its querying

### ISSUE THREAD

**Issue by qalqi** [julianburr/react-firebase#3]
react-firebase-context doesn't seem to support the firebase real-time database.
Can you please add it features roadmap along with examples

Thanks

**Comment by julianburr** [comment]
Hey, can you explain what you mean? It supports firestore, which is the new "version". Do you mean the old realtime database version?

**Comment by qalqi** [comment]
Yeah.  There are two different databases serving two different purposes under firebase services.
1) Realtime Database ( Currently Not supported with react-firebase-context)
2) Cloud Firestore ( Currently supported with react-firebase-context )

[Firebase offers two cloud-based](https://firebase.google.com/docs/database/rtdb-vs-firestore)  client-accessible database solutions that support realtime data syncing

**Comment by julianburr** [comment]
Gotcha, I'm not actively planning to implement that, but I'll leave the issue open for anyone else to contribute and add it :blush:
